systems/subscriber/subscriber-registry/pb/subscriber.proto

```diff
@@ -0,0 +1,190 @@
+// networkID field is used to specify the network id for getting subscribers
+string networkID = 1 [(validator.field) = {uuid_ver: 0, string_not_empty: true}, json_name = "network_id"];
+
+}
+
+// GetByNetworkResponse message is used to get all subscribers by network id
+message GetByNetworkResponse{
+// Repeated field of Subscriber message
+repeated Subscriber subscribers = 1;
+}
+
+// DeleteSubscriberResponse message is used to delete a subscriber
+message DeleteSubscriberResponse{
+// subscriberID field is used to specify the subscriber id for deletion
+string subscriberID = 1;
+
+}
+
+// GetSubscriberRequest message is used to get a subscriber by subscriber ID
+message GetSubscriberRequest{
+// subscriberID field is used to specify the subscriber id for getting
+string subscriberID = 1 [(validator.field) = {uuid_ver: 0, string_not_empty: true}, json_name = "subscriber_id"];
+}
+
+// GetSubscriberResponse message is used to get a subscriber by subscriber ID
+message GetSubscriberResponse{
+// Subscriber field contains the subscriber information
+Subscriber subscriber = 1;
+}
+
+message AddSubscriberRequest {
+    // first name of the subscriber, must not be empty and have a length greater than 1
+    string firstName = 1 [(validator.field) = { string_not_empty: true, length_gt: 1 }];
+    // last name of the subscriber, must not be empty and have a length greater than 1
+    string lastName = 2 [(validator.field) = { string_not_empty: true, length_gt: 1 }];
+    // email of the subscriber, must be in email format
+    string email = 3  [(validator.field) = { human_error:"must be an email format" regex: "^$|^[a-z0-9._%+\\-]+@[a-z0-9.\\-]+\\.[a-z]{2,4}$"}];
+    // phone number of the subscriber, must be in phone number format
+    string phoneNumber = 4  [(validator.field) = { human_error:"must be a phone number format", regex: "^$|^(?:(?:\\(?(?:00|\\+)([1-4]\\d\\d|[1-9]\\d?)\\)?)?[\\-\\.\\ \\\\\\/]?)?((?:\\(?\\d{1,}\\)?[\\-\\.\\ \\\\\\/]?){0,})(?:[\\-\\.\\ \\\\\\/]?(?:#|ext\\.?|extension|x)[\\-\\.\\ \\\\\\/]?(\\d+))?$"}];
+    // address of the subscriber
+    string address = 5;
+    // idSerial of the subscriber
+    string idSerial =6;
+    // network ID of the subscriber, must be a UUID and not empty
+    string networkID = 7 [(validator.field) = {uuid_ver: 0, string_not_empty: true}, json_name = "network_id"];
+    // proof of identification of the subscriber, must not be empty and have a length greater than 1
+    string proofOfIdentification =8 [(validator.field) = { string_not_empty: true, length_gt: 1 }];
+    // date of birth of the subscriber
+    google.protobuf.Timestamp dateOfBirth = 9;
+    // gender of the subscriber, must not be empty and have a length greater than 1
+    string gender =10 [(validator.field) = { string_not_empty: true, length_gt: 1 }];
+    // org ID of the subscriber, must be a UUID and not empty
+    string orgID = 11 [(validator.field) = {uuid_ver: 0, string_not_empty: true}, json_name = "org_id"];
```

Comment:
If we are keeping both the networkID and the orgID, then we need to make sure that these two information never mismatch.